ports/nrf/boards/feather52840/README.md

```diff
@@ -174,28 +169,48 @@ BUTTON1 still pressed as you come out of reset).
 This will give you a **fast blinky DFU pattern** to indicate you are in DFU
 mode.
 
-At this point, you can build and flash a CircuitPython binary via the following
+At this point, you can **build and flash** a CircuitPython binary via the following
 command:
 
 ```
-$ make V=1 SD=s140 SERIAL=/dev/tty.usbmodem1411 BOARD=feather52840 dfu-gen dfu-flash
+$ make V=1 SD=s140 SERIAL=/dev/tty.usbmodem1411 BOARD=feather52840 all dfu-gen dfu-flash
 ```
 
 This should give you the following results:
 
 ```
-$ make V=1 SD=s140 SERIAL=/dev/tty.usbmodem1411 BOARD=feather52840 dfu-gen dfu-flash
+$make V=1 BOARD=feather52840 SD=s140 SERIAL=/dev/tty.usbmodem1411 dfu-gen dfu-flash
 nrfutil dfu genpkg --sd-req 0xFFFE --dev-type 0x0052 --application build-feather52840-s140/firmware.hex build-feather52840-s140/dfu-package.zip
 Zip created at build-feather52840-s140/dfu-package.zip
-nrfutil --verbose dfu serial --package build-feather52840-s140/dfu-package.zip -p /dev/tty.usbmodem1411 -b 115200
-Upgrading target on /dev/tty.usbmodem1411 with DFU package /Users/kevintownsend/Dropbox/microBuilder/Code/CircuitPython/circuitpython-mb/ports/nrf/build-feather52840-s140/dfu-package.zip. Flow control is disabled.
-Starting DFU upgrade of type 4, SoftDevice size: 0, bootloader size: 0, application size: 195252
+nrfutil --verbose dfu serial --package build-feather52840-s140/dfu-package.zip -p /dev/ttyACM1 -b 115200 --singlebank
+Upgrading target on /dev/ttyACM1 with DFU package /home/hathach/Dropbox/adafruit/circuitpython/ada_cp/ports/nrf/build-feather52840-s140/dfu-package.zip. Flow control is disabled, Single bank mode
```

Comment:
Not sure if this change is relevant, changes port and path

Reply:
nrfutil 0.52d has the new --singlebank option, it just happens that my copied text is different than ktown